test/exceptions/Makefile

```diff
@@ -94,12 +96,14 @@ $(ROOT)/refcounted.done: $(ROOT)/refcounted
 	$(QUIET) $<
 	@touch $@
 
-ifeq (ldmd,$(findstring ldmd,$(HOST_DC)))
+ifeq (ldmd,$(findstring ldmd,$(DMD)))
 # LDC: Make sure allocation intended to provoke exception is not elided.
 $(ROOT)/invalid_memory_operation: DFLAGS+=-disable-gc2stack
 endif
 $(ROOT)/unittest_assert: DFLAGS+=-unittest -version=CoreUnittest
 $(ROOT)/line_trace: DFLAGS+=$(LINE_TRACE_DFLAGS)
+$(ROOT)/long_backtrace_trunc: DFLAGS+=$(LINE_TRACE_DFLAGS)
+$(ROOT)/rt_trap_exceptions: DFLAGS+=$(LINE_TRACE_DFLAGS)
```

Comment:
These 2 are required for LDC, which doesn't include `-L--export-dynamic` by default like DMD.